src/containers/SignUp.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+import NavBar from '../components/NavBar';
+import SignupForm from '../components/SignupForm';
+import Footer from '../components/Footer';
+
+class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <NavBar />
+        <SignupForm />
+        <Footer />
+      </React.Fragment>
+    );
```

Comment:
You need to put the components such as footer which is rendered in every page in the App component. This will help to eliminate the need to put it in every component you make.

Reply:
@hoslack, Thank you for the recommendation.
The footer for the homepage is different from the footer for the other pages.